### PR TITLE
Feature/rollups alphabetize fg fields

### PR DESCRIPTION
**Pull request by randi274**
# Critical Changes

# Changes
- Fix for W-024781 to alphabetize filter rule fields. 
- Fixed FG name + description and FR popup not showing required icon in edit. 
- Fixed bug where number amount was not required to save FR (other than when operator is equals || notEquals).
- Fixed top padding issue for Value textarea by explicitly hiding all label markup when input has a custom tooltip.

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/aura/CRLP_FilterGroup/CRLP_FilterGroup.cmp [hunk 1]
```diff
@@ -270,7 +270,7 @@
                                                      required="true" label="{!v.labels.constant}" type="datetime-local"/>
                                 </aura:if>
                                 <aura:if isTrue="{!equals(v.filterRuleFieldType, 'number')}">
-                                    <lightning:input class="slds-p-around_small" value="{!v.activeFilterRule.value}"
+                                    <lightning:input aura:id="filterRuleField" class="slds-p-around_small" value="{!v.activeFilterRule.value}"
```
Comment (force2b) [inline src/aura/CRLP_FilterGroup/CRLP_FilterGroup.cmp]: Looks like this breaks the recent change to create a default filter rule for `Amount=null` to mimic legacy behavior.
image

**Comment by randi274** [comment]
@force2b these changes are now ready with the changes from feature/rollups__fg-req-value-fix merged into it (so they should be merged in together, preferably).

**Comment by force2b** [comment]
Thanks, though I don't see a PR for the `feature/rollups__fg-req-value-fix` branch?

**Comment by randi274** [comment]
@bethbrains is the feature/rollups__fg-req-value-fix branch ready for a PR? This PR is now tied with the changes made in that branch, to make sure the FG items are ready together.

**Comment by bethbrains** [comment]
Done! https://github.com/SalesforceFoundation/Cumulus/pull/3227

**Pull request merged**